### PR TITLE
question endpoints configured

### LINKED ISSUE #1: Create /GET Endpoints for retrieving questions

### LINKED ISSUE #2: Create /POST endpoint for creating questions
Acceptance Criteria:
- [x] Returns a 200 OK when a question is created
- [x] Ships information to logs when there are any non 200 responses
- [x] Publicly exposes endpoint should be POST to /questions

### PR DESCRIPTION
**Pull request by aakano684**

### CODE REVIEW

File: src/main/resources/application.properties
Comment (noce2cicd) [inline src/main/resources/application.properties]: @overlxrd , is there some script you used to startup the local database and set its password to this? If so, it would be worth adding that into this PR as well.

File: .classpath
Comment (noce2cicd) [inline .classpath:1]: I can't remember, did we say we would start/stop adding .classpaths into version control?

File: README.md
Comment (noce2cicd) [inline README.md]: I guess we need to modify this line so it shows we get and retrieve questions for our API?

File: src/main/java/io/mathdojo/QuestionHandler.java
Comment (noce2cicd) [inline src/main/java/io/mathdojo/QuestionHandler.java]: It would be good to have this logging uncommented and modified for our use case. That way as we debug, we can be sure the code definitely arrived here.

File: src/main/java/io/mathdojo/model/Question.java
Comment (noce2cicd) [inline src/main/java/io/mathdojo/model/Question.java]: Why do we have a Question constructor that takes in Question objects?

File: src/test/java/io/mathdojo/QuestionFunctionTest.java
Comment (noce2cicd) [inline src/test/java/io/mathdojo/QuestionFunctionTest.java:15]: We need to modify this so that we run some unit tests.

**Review by noce2cicd — CHANGES_REQUESTED** [review]
Looks good, 2 things to bear in mind before we merge this:
1. As is, since the addtion of commit 724a112 although it might deploy the function correctly, none of the endpoints will work (as it'll reach for a database that doesn't exist). We could either revert the commit where we add local db config (and move it to another branch where we can continue the db integration work) or not merge this at all till the cloud db integration is done as well.

2. We need to have a few unit tests for the functionality we've got now. Not doing this will bite us hard in the future.

File: src/main/java/io/mathdojo/model/Question.java
Comment (aakano684) [reply src/main/java/io/mathdojo/model/Question.java]: I think I was testing something. I've removed that

File: src/main/java/io/mathdojo/QuestionHandler.java
Comment (aakano684) [reply src/main/java/io/mathdojo/QuestionHandler.java]: done

File: src/main/resources/application.properties
Comment (aakano684) [reply src/main/resources/application.properties]: The database config has since been changed. Details on running locally will be in the README

File: .classpath
Comment (aakano684) [reply .classpath:1]: I think we concluded that it's irrelevant but It's probably best practice to avoid doing that.

File: README.md
Comment (aakano684) [reply README.md]: yes, I'll attempt to do that

File: src/test/java/io/mathdojo/QuestionFunctionTest.java
Comment (aakano684) [reply src/test/java/io/mathdojo/QuestionFunctionTest.java:15]: I modified them and will add more

**Review by noce2cicd — APPROVED** [review]

**Pull request merged**
